src/main/java/sg/ncl/MainController.java

```diff
@@ -3410,4 +3408,28 @@ private String getUsageStatisticsByTeamId(String id) {
         }
         return response.getBody().toString();
     }
+
+    private String checkUserException(ExceptionState exceptionState, MyErrorResource error) {
+        if (exceptionState == USER_ID_NULL_OR_EMPTY_EXCEPTION || exceptionState == USER_NOT_FOUND_EXCEPTION) {
+            return error.getMessage();
+        }
+        else return null;
+
+    }
+
+    private String checkTeamException(ExceptionState exceptionState, MyErrorResource error) {
+        if (exceptionState == TEAM_NAME_NULL_OR_EMPTY_EXCEPTION || exceptionState == TEAM_NAME_NULL_OR_EMPTY_EXCEPTION || exceptionState == TEAM_NOT_FOUND_EXCEPTION) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Identical sub-expressions on both sides of operator "||" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1764)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)